pombola/writeinpublic/forms.py

```diff
@@ -0,0 +1,28 @@
+from django import forms
+from django.forms import SelectMultiple, ModelMultipleChoiceField
+
+from pombola.core.models import Person
+
+from .client import WriteInPublic
+
+
+class RecipientForm(forms.Form):
+    persons = ModelMultipleChoiceField(queryset=None)
+
+    def __init__(self, *args, **kwargs):
+        super(RecipientForm, self).__init__(*args, **kwargs)
+        self.fields['persons'].queryset = Person.objects.filter(
+            identifiers__scheme='everypolitician',
+            identifiers__identifier__isnull=False,
+        )
```

Comment:
I think I'd add a comment here explaining why you're setting the queryset dynamically. (I think I see why, but I had to think about it quite hard!)

Reply:
:+1: I've added a comment about this in https://github.com/mysociety/pombola/pull/2345/commits/04950200b8af7dbf970163f88e8b560b77b0043e.